src/pcap/api.go

```diff
@@ -98,9 +94,19 @@ func (api *API) Status(context.Context, *StatusRequest) (*StatusResponse, error)
 	return apiStatus, nil
 }
 
-// handlerRegistered checks if handler is registered.
+func (api *API) RegisteredResolverNames() []string {
+	resolverNames := make([]string, len(api.resolvers))
+	i := 0
+	for key := range api.resolvers {
+		resolverNames[i] = key
+		i++
+	}
+	return resolverNames
+}
+
+// resolverRegistered checks if handler is registered.
 // returns false, if the handler is not registered.
-func (api *API) handlerRegistered(handler string) *bool {
+func (api *API) resolverRegistered(handler string) *bool {
```

Comment:
I am not sure the terms handler and resolver in the api.go are used in the consistent way.